Network Page: Fixing issue #305 and #295

dongbohu opened the pull request:
This simple PR is for issue #305 and #295.

dongbohu commented:
Here is a demo page:
http://165.123.67.202/#/gene_network/?genes=6552&mlmodel=4
Click one of the edges whose popup window has a clickable signature name, and a new tab should show up, but the original tab is still there.

dongbohu commented:
@mhuyck: I added another commit to address issue #295 in this PR too.

mhuyck commented on interface/src/app/help/help.tpl.html:
```diff
@@ -52,7 +52,14 @@ <h3><u>Genes</u></h3>
+    background correction so that the view prioritizes the genes that are most
+    connected to the query genes. (If the correction were applied, the view
+    would prioritize genes that are more connected to the query genes than to
+    other genes.)
```
This is a great explanation. Thank you for nailing it down!

A couple of things will help make this section more readable:

1. This **Genes** section, unlike all others on this page, has three distinct paragraphs rather than just one. They should each be formatted with `<p>...</p>` instead of having `<br>` to allow the browser to show that separation more clearly. In fact, the rest of the page sections should be corrected to use `<p>` tags rather than just bare text after the headers.
2. The explanation in this last part is now clear to me, but the difference between corrected _vs._ uncorrected ranking could be made still more clear with a little emphasis added, I think:

> Some genes are connected to many genes while others are connected to fewer
   genes. Right now, the genes in the network view are ranked <em>without</em>
   background correction so that the view prioritizes the genes that are most
   connected to the query genes. (If the correction were applied, the view
   would prioritize genes that are <em>more connected</em> to the query genes 
   than they are to other genes.)

dongbohu commented:
@mhuyck I like your comments about adding `<p>` tags in `Genes` section. I have added them in this section, but I didn't add it in the other sections, because so far they only have one paragraph anyway. 

I also followed your comment on making some keywords italic, and removed the extra `<br>` at the end of each section to make the page more compact. Please let me know what you think.

mhuyck commented:
LGTM. Thanks, @dongbohu!
